enable/gcbench/publish.py

```diff
@@ -108,10 +114,8 @@ def _build_comparison_table(backends, comparisons):
         link = f'<a href="{name}.html">'
         row = [f"<td>{link}{name}</a></td>"]
         for bend in backends:
-            # Each backend stat includes a "valid" flag
-            stat, valid = stats[bend]
-            # Which gets used to add a CSS class for styling
-            klass = "valid" if valid else "invalid"
+            # Each backend stat includes a CSS class for table styling
+            stat, klass = stats[bend]
```

Comment:
maybe this stats could be called something different in the for loop above?  It is the stats after they have been run though a formatting function eg `_format_benchmark`.  

It is pretty inconsequential, but the code names are a little confusing as was thinking `stats` was a dict of a different form at first